src/components/Comments/WriteComment.js

```diff
@@ -138,11 +138,7 @@ const WriteComment = ({
           </TouchableOpacity>
         ) : (
           <View style={styles.isViewOnlyIcon}>
-            <SendIcon
-              fillBackground={
-                !isCommentEnabled || loadingUser ? COLORS.gray : COLORS.signed_primary
-              }
-            />
+            <SendIcon type={CHAT_SIGNED} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code removes the condition that changes the button color based on whether commenting is enabled or if the user data is still loading. This could potentially lead to a situation where the send button appears active (with the `CHAT_SIGNED` color) even when it's not supposed to be, such as when the user data is still loading or commenting is disabled. I suggest reintroducing this condition to ensure the button color accurately reflects its state.

```diff
- <SendIcon type={CHAT_SIGNED} />
+ <SendIcon type={!isCommentEnabled || loadingUser ? CHAT_DISABLED : CHAT_SIGNED} />
```
Please replace `CHAT_DISABLED` with the appropriate constant for the disabled state color.


<!-- This is an auto-generated comment by OSS CodeRabbit -->